django/contrib/auth/views.py

```diff
@@ -133,12 +133,21 @@ def get_context_data(self, **kwargs):
 
 
 @deprecate_current_app
-def login(request, *args, **kwargs):
+def login(request, template_name='registration/login.html', redirect_field_name=REDIRECT_FIELD_NAME,
+          authentication_form=AuthenticationForm, extra_context=None,
+          redirect_authenticated_user=False):
     warnings.warn(
         'The login() view is superseded by the class-based LoginView().',
         RemovedInDjango21Warning, stacklevel=2
     )
-    return LoginView.as_view(**kwargs)(request, *args, **kwargs)
+
+    return LoginView.as_view(
+        template_name = template_name,
+        redirect_field_name=redirect_field_name,
+        form_class=authentication_form,
+        extra_context=extra_context,
+        redirect_authenticated_user=redirect_authenticated_user
```

Comment:
Include a trailing comma in cases like this so that if more items are added later, this line doesn't have to be modified again.